packages/data-layer/src/data-layer.test.ts

```diff
@@ -492,10 +543,80 @@ describe("passport verification", () => {
   });
 });
 
+describe("projects retrieval", () => {
+  test("can retrieve project by id", async () => {
+    const fetchMock = vi.fn().mockResolvedValue({
+      status: 200,
+      headers: new Headers({ "content-type": "application/json" }),
+      json: async () => ({
+        project: mockProject,
+      }),
+    });
+
+    const dataLayer = new DataLayer({
+      fetch: fetchMock,
+      search: { baseUrl: "https://example.com" },
+      subgraph: { endpointsByChainId: {} },
+      indexer: { baseUrl: "https://indexer-staging.fly.dev/graphql" },
+    });
+
+    const project = await dataLayer.getProjectById({
+      projectId:
+        "0x8a79249b63395c25bd121ba6ff280198c399d4fb3f951fc3c42197b54a6db6a6",
+      chainId: 11155111,
+      alloVersion: "allo-v2",
+    });
+
+    // todo: update to test the entire object when the missing fields are added
```

Comment:
what do we need to do here in the indexer? those fields should be in the metadata right? or do we want to extract them?

Reply:
They should be in the metadata but are not returned.